dkist/io/asdf/resources/manifests/dkist-1.2.0.yaml

```diff
@@ -0,0 +1,14 @@
+%YAML 1.1
+---
+id: asdf://dkist.nso.edu/dkist/manifests/dkist-1.2.0
+extension_uri: asdf://dkist.nso.edu/dkist/extensions/dkist-1.0.0
+title: DKIST extension
+description: ASDF schemas and tags for DKIST classes.
+
+tags:
+  - schema_uri: "asdf://dkist.nso.edu/schemas/file_manager-1.0.0"
+    tag_uri: "asdf://dkist.nso.edu/tags/file_manager-1.0.0"
+  - schema_uri: "asdf://dkist.nso.edu/schemas/dataset-1.2.0"
+    tag_uri: "asdf://dkist.nso.edu/tags/dataset-1.2.0"
```

Comment:
@braingram When making a new manifest like this, should it have the old tags in it as well or is it better to leave those to the old manifests?

Reply:
In this case it looks like it's better to leave those in the old manifests and not include them in the new one. As long as the old manfiests/extensions are registered (after the new extension) asdf will use whatever old manifest/extension contains the old tag when it reads a file with that tag. On write it will use the first extension that supports the type (which is why the new extension should be registered first). If you were to include the old tags you would need to update `select_tag` in the `Converter` to pick the appropriate tag (either the new or old version). Otherwise including the old tags would result in writing only the old tags.

Reply:
Thanks!